app/tests/spec/lib/fxa-client.js

```diff
@@ -646,6 +646,180 @@ define(function (require, exports, module) {
       });
     });
 
+    describe('sessionReauth', function () {
```

Comment:
nit: I see a mix of ES6 syntax, might be a good opportunity to update new code.

Reply:
Thanks; I guess you can tell the bits that I copy-pasted from other tests vs the bits I wrote out from scratch :-)